django/db/models/base.py

```diff
@@ -324,7 +326,7 @@ def _prepare(cls):
 
             # defer creating accessors on the foreign class until we are
             # certain it has been created
-            def make_foreign_order_accessors(field, model, cls):
+            def make_foreign_order_accessors(cls, model, field):
```

Comment:
The old API takes ``field, model, cls`` – the new one takes n models, and optional keyword arguments. Field is passed as a keyword argument, so it has to appear last.